### PR TITLE
DOCS-545: Yandex Datalens data analytics and visualization tool

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation

### CODE REVIEW

File: docs/en/interfaces/third-party/gui.md [hunk 1]
```diff
@@ -101,6 +101,22 @@ Features:
 - Refactorings.
 - Search and Navigation.
 
+### Yandex DataLens {#yandex-datalens}
+
+[Yandex DataLens](https://cloud.yandex.ru/services/datalens) is a service of data visualization and analytics.
+
+Features:
+
+- Creation of simple visualizations and complex dashboards.
```
Comment (qoega) [inline docs/en/interfaces/third-party/gui.md]:
```suggestion
- Supports simple visualizations and complex dashboards.
```
But I don't like both. First point should be more descriptive...

File: docs/en/interfaces/third-party/gui.md [hunk 2]
```diff
@@ -101,6 +101,22 @@ Features:
...
+- Public dashboards.
+- Support for multiple data sources.
```
Comment (blinkov) [inline docs/en/interfaces/third-party/gui.md]:
```suggestion
- Support for multiple data sources including ClickHouse.
```

File: docs/en/interfaces/third-party/gui.md [hunk 3]
```diff
@@ -101,6 +101,22 @@ Features:
...
+- Storage for materialized data.
```
Comment (blinkov) [inline docs/en/interfaces/third-party/gui.md]:
```suggestion
- Storage for materialized data based on ClickHouse.
```

File: docs/en/interfaces/third-party/gui.md (hunk 1 above)
Comment (blinkov) [inline docs/en/interfaces/third-party/gui.md]:
```suggestion
- Wide range of available visualisations, from simple bar charts to complex dashboards.
```

File: docs/en/interfaces/third-party/gui.md [hunk 4]
```diff
@@ -101,6 +101,22 @@ Features:
...
 - Search and Navigation.
 
+### Yandex DataLens {#yandex-datalens}
+
+[Yandex DataLens](https://cloud.yandex.ru/services/datalens) is a service of data visualization and analytics.
+
+Features:
+
+- Creation of simple visualizations and complex dashboards.
+- Public dashboards.
```
Comment (blinkov) [inline docs/en/interfaces/third-party/gui.md]:
```suggestion
- Dashboards could be made publicly available.
```

File: docs/en/interfaces/third-party/gui.md [hunk 5]
```diff
@@ -101,6 +101,22 @@ Features:
...
+- Support for multiple data sources.
+- Storage for materialized data.
+
+For low-load projects, DataLens is [available for free](https://cloud.yandex.com/docs/datalens/pricing), including for commercial use.
```
Comment (blinkov) [inline docs/en/interfaces/third-party/gui.md]:
```suggestion
DataLens is [available for free](https://cloud.yandex.com/docs/datalens/pricing) for low-load projects, even for commercial use.
```

File: docs/en/interfaces/third-party/gui.md [hunk 6]
```diff
@@ -101,6 +101,22 @@ Features:
...
+
+- [DataLens documentation](https://cloud.yandex.com/docs/datalens/).
+- [Visualizing data from a ClickHouse database](https://cloud.yandex.com/docs/datalens/solutions/data-from-db-visualization) tutorial.
```
Comment (blinkov) [inline docs/en/interfaces/third-party/gui.md]:
```suggestion
- [Tutorial](https://cloud.yandex.com/docs/datalens/solutions/data-from-db-visualization) on visualizing data from a ClickHouse database.
```

**Review by blinkov — CHANGES_REQUESTED** [review]

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**